drivers/SmartThings/matter-sensor/src/init.lua

```diff
@@ -15,13 +15,26 @@
 local capabilities = require "st.capabilities"
 local log = require "log"
 local clusters = require "st.matter.clusters"
+local im = require "st.matter.interaction_model"
 local MatterDriver = require "st.matter.driver"
 local utils = require "st.utils"
 
 local BATTERY_CHECKED = "__battery_checked"
+local BOUNDS_CHECKED = "__bounds_checked"
+local TEMP_BOUND_RECEIVED = "temp_bound_received"
+local TEMP_MIN = "__temp_min"
+local TEMP_MAX = "__temp_max"
 
 local HUE_MANUFACTURER_ID = 0x100B
 
+local function get_field_for_endpoint(device, field, endpoint)
+  return device:get_field(string.format("%s_%d", field, endpoint))
+end
+
+local function set_field_for_endpoint(device, field, endpoint, value, persist)
+  device:set_field(string.format("%s_%d", field, endpoint), value, {persist = persist})
+end
```

Comment:
Can we add these to `Device` in the lua libraries as a follow-up?

Reply:
Good thinking - created a ticket for this here: https://smartthings.atlassian.net/browse/CHAD-12925